src/main/resources/handlebars/soknad.hbs

```diff
@@ -131,6 +131,16 @@
         <img class="checkbox" src="{{ image "Checkbox_on.png" }}"/> Nei
     {{/if}}
 
+    <!-- Info om har ikke lastet opp vedlegg -->
+    {{#if harIkkeLastetOppLegeerklæring}}
```

Comment:
Dette her viser på `Legeerklæring ` og `Ingen vedlegg er lastet opp.`
Kan fikses slik:
```hbs
{{#if harLastetOppLegeerklæring}}
 <h2>
        <span>
            Legeerklæring
        </span>
        </h2>
{{else}}
        <p>Ingen vedlegg er lastet opp.</p>
{{/if}}
```

Reply:
Jira oppgaven sier "Skal altså **kun** komme på PDF hvis det ikke er lastet opp noe."

Reply:
Aha sorry, my bad. Misforstod hva  som stod her. Det er isåfall korrekt 👍